applications/tube_network/src/statistics.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2018 Grakn Labs Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import grakn
+import tube_network.settings as settings
+
+def print_to_log(title, content):
+  print(title)
+  print("")
+  print(content)
+  print("\n")
+
+# How many stations do exist?
+def execute_query_1(question, transaction):
```

Comment:
These functions should be named as per the comments, to describe their functions, maybe prefix them all with _query_ to indicate they're part of a family

Reply:
very good point!